src/System.Private.ServiceModel/tests/Common/src/project.lock.json

```diff
@@ -367,15 +345,13 @@
           "lib/DNXCore50/System.Net.Security.dll": {}
         }
       },
-      "System.Net.Sockets/4.0.10-beta-23024": {
+      "System.Net.Sockets/4.0.0-beta-23024": {
```

Comment:
This downgrade to an earlier version of System.Net.Sockets is further evidence we want an explicit dependency in project.json.  I see that version '0' actually passes in NET Native but fails in CoreCLR.  I believe CoreCLR needs the '10' version.